### ISSUE TITLE
Add tuple conversion support

### ISSUE THREAD

**Issue by alex88** [balance-platform/pillar#40]
Would it make sense to add elixir to clickhouse tuple support?
I was able to get it working locally by adding

```elixir
  def convert(param) when is_tuple(param) do
    values = param
    |> Tuple.to_list()
    |> Enum.map_join(",", &convert/1)

    "(#{values})"
  end
```

**Comment by sofakingworld** [comment]
If there is no such check, then there is no problem.

Would you add this conversion?

**Comment by sofakingworld** [comment]
Is solving in https://github.com/balance-platform/pillar/pull/63

**Comment by sofakingworld** [comment]
Solved and added in 0.36.0
